apps/staking/package.json

```
@@ -11,12 +11,14 @@
   },
   "dependencies": {
     "@hookform/resolvers": "^3.5.0",
+    "@session/auth": "workspace:*",
     "@session/contracts": "workspace:*",
     "@session/sent-staking-js": "workspace:*",
     "@session/ui": "workspace:*",
     "@session/util": "workspace:*",
     "@session/wallet": "workspace:*",
     "@tanstack/react-query": "^5.32.1",
+    "better-sqlite3-multiple-ciphers": "^11.1.2",
```

Suggest we have a fixed version here for security's sake. We don't want it to upgrade on it's own.
```suggestion
    "better-sqlite3-multiple-ciphers": "11.1.2",
```

---

fixed